lib/AFS/CellCC/Util.pm

```
@@ -401,4 +401,18 @@ calc_checksum($$$$$$) {
     return "$algo:$checksum";
 }
 
+# Get the value of the field provided as an argument for every job and return
+# these values as an array.
+sub
+get_jobs_info($@) {
+    my ($field, @jobs) = @_;
+    my @info;
+    for my $job (@jobs) {
+        if (exists $job->{$field}) {
```

I don't think you want to silently ignore non-existent fields. Did you see this actually happen? It seems like that would be a serious error, for the places this is used.